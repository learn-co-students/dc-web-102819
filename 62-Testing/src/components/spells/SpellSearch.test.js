import React from 'react'
import SpellSearchBar from './SpellSearch'
import { shallow } from 'enzyme'

it("Renders the SpellSearchBar", () => {
  const renderedCard = shallow(<SpellSearchBar spellSearch={() => {}}/>);
  expect(renderedCard).toMatchSnapshot();

});

it('Is able to submit form', () => {
    const renderedCard = shallow(<SpellSearchBar spellSearch={() => {}} />);
    renderedCard.find("input").simulate("submit");
    
})