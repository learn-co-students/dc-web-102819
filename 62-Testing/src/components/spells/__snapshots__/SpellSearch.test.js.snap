// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the SpellSearchBar 1`] = `ShallowWrapper {}`;
